index.js

```diff
@@ -188,5 +188,12 @@ const saveEditChanges = (event) => {
 
 };
 
+export const valueCheck = (num1, num2) => {
+  if (num1 > num2) {
+    return "Num1 is Bigger then Num2";
+  }
+  return "Num1 is Smaller then Num2";
+}
```

Comment:
**Suggestion:** The return strings of the `valueCheck` function are not very descriptive. Consider making them more informative by including the actual values of `num1` and `num2`. [enhancement]
```suggestion
export const valueCheck = (num1, num2) => {
  if (num1 > num2) {
    return `${num1} is bigger than ${num2}`;
  } else if (num1 < num2) {
    return `${num1} is smaller than ${num2}`;
  } else {
    return `${num1} is equal to ${num2}`;
  }
}
```